.eslintrc.json

```diff
@@ -1,14 +1,19 @@
 {
+    "env": {
+        "browser": true
+    },
     "extends": [
-        "airbnb",
+        "airbnb-base",
         "prettier"
     ],
+    "parser": "babel-eslint",
     "plugins": [
         "prettier"
     ],
     "rules": {
-        "prettier/prettier": [
-            "error"
-        ]
+        "import/prefer-default-export": "warn",
+        "no-param-reassign": "warn",
```

Comment:
set these to warnings because we should still avoid them but I don't want them to make our CI fail